keras_cv/bounding_box/utils.py

```diff
@@ -87,3 +87,73 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def pad_with_sentinels(bounding_boxes):
+    """Pads the given bounding boxes to convert it from a ragged tensor to
+    a regular tensor.
+
+    Args:
+      bounding_boxes: a ragged tensor of bounding boxes in 'corners' format.
+      Can be batched or unbatched.
+
+    Returns:
+      bounding_boxes_tensor: a tensor containing the -1 padded bounding boxes.
+    """
+    return bounding_boxes.to_tensor(-1)
+
+
+def filter_sentinels(bounding_boxes):
+    """filters the -1 padded bounding boxes to convert it from a regular
+       tensor to a ragged tensor.
+
+    Args:
+      bounding_boxes: a tensor of bounding boxes in 'corners' format.
+      Can be batched or unbatched.
+
+    Returns:
+      bounding_boxes_tensor: a ragged tensor containing filtered bounding boxes.
+    """
+    if isinstance(bounding_boxes, tf.Tensor):
+        tf.RaggedTensor.from_tensor(bounding_boxes)
```

Comment:
this is a no-op, there is no assignment here

Reply:
fixed it.